packages/mirage-fs-unix/mirage-fs-unix.1.1.0/opam

```diff
@@ -12,5 +13,6 @@ depends: [
   "mirage-types" {>= "1.1.0"}
   "ocamlbuild" {build}
 ]
-ocaml-version: [>= "4.00.0"]
+available: [ ocaml-version >= "4.00.0" ]
 dev-repo: "git://github.com/mirage/mirage-fs-unix"
+conflicts: [ "mirage-fs-unix" {="1.0.0"} ]
```

Comment:
it doesn't work anyways (since at the time of conflict evaluation mirage-fs-unix is not installed)... I think it is better to say `available: ocaml-version < 0` (as done in the subsequent force-pushed commit)